### PR TITLE
docs(manual): Quick fix for set language hint feature

### PR DESCRIPTION
**Pull request by avinashbest**
- Change the named keyboard apps to they need to support the setImeHintLocales Android API

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -1056,8 +1056,8 @@ Notifications only work for top level decks. Please let us know if you want us t
 === Automatic Language Selection
 AnkiDroid has an in Automatic Language Selection feature in the Note Editor starting with AnkiDroid 2.13. This feature allows you to define a default language to be used in your keyboard for a field in a note type.
 
-For example, if you have Russian and English note fields, the keyboard layout (in keyboards like SwiftKey and Swype) will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
 
+For example, if you have Russian and English note fields, the keyboard layout (need to support the setImeHintLocales Android API) will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```
Comment (mikehardy) [inline manual.asc:1060]:
```suggestion
For example, if you have Russian and English note fields, and your keyboard supports the setImeHintLocales Android API, the keyboard layout will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```

**Review by mikehardy — APPROVED** [review]
thank you!

**Comment by avinashbest** [comment]
Anytime, as ready for the community

**Comment by mikehardy** [comment]
Hi there!

Just a friendly notice (one per person for PRs merged in March, regardless of PR quantity) that we try to process OpenCollective payments monthly - it's time for March 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

Thanks!

**Pull request merged**